docs/source/conf.py

```diff
@@ -377,6 +385,7 @@ def generate_weights_table(module, table_name, metrics, include_patterns=None, e
     with open(generated_dir / f"{table_name}_table.rst", "w+") as table_file:
         table_file.write(".. rst-class:: table-weights\n")  # Custom CSS class, see custom_torchvision.css
         table_file.write(".. table::\n")
+        table_file.write("    :class: datatable\n")
```

Comment:
Would it work to use the `.. rst-class` directive as above? If not, do you know why?

Reply:
Works fine